certbot/storage.py

```diff
@@ -214,6 +214,23 @@ def get_link_target(link):
         target = os.path.join(os.path.dirname(link), target)
     return os.path.abspath(target)
 
+def _write_live_readme_to(readme_path):
+    with open(readme_path, "w") as f:
+        logger.debug("Writing README to %s.", readme_path)
+        f.write("This directory contains your keys and certificates.\n\n"
+                "`privkey.pem`  : the private key for your certificate.\n"
+                "`fullchain.pem`: the certificate file used in most server software.\n"
+                "`chain.pem`    : used for OCSP stapling in Nginx >=1.3.7.\n"
+                "`cert.pem`     : will break many server configurations, and "
+                                    "should not be used\n"
+                "                 without reading further documentation (see link below).\n\n"
+                "WARNING: DO NOT MOVE THESE FILES!\n"
```

Comment:
I wonder if this ought to be even more strongly worded or more general, like "DO NOT MOVE **OR RENAME** THESE FILES".